src/js/guestModules/checkedForm.js

```diff
@@ -0,0 +1,152 @@
+const eyePng = require('../../icons/eye.png');
+
+const { $id } = require('../utils');
+const { createOpenQuestion, createClosedQuestion, createNumberQuestion } = require('../common/form');
+
+const CheckedForm = {
+    initialized: false,
+
+    init() {
+        this.initialized = true;
+        this.assignEventListeners();
+
+        // TODO: Bazka
+        const str = '[{"formTitle":"Nowy","owner":"metin90.1998","hrEmployer":"metin90.1998","points":[true,false, false]},{"formTitle":"Nowy","owner":"metin90.1998","hrEmployer":"metin90.1998","points":[true,true]},{"formTitle":"Nowy","owner":"metin90.1998","hrEmployer":"metin90.1998","points":[true,true]},{"formTitle":"Nowy","owner":"metin90.1998","hrEmployer":"metin90.1998","points":[true,true]},{"formTitle":"Nowy","owner":"metin90.1998","hrEmployer":"metin90.1998","points":[true,true]},{"formTitle":"Nowy","owner":"metin90.1998","hrEmployer":"metin90.1998","points":[false,true]}]';
+        const forms = JSON.parse(str);
+
+        for (const [it, form] of forms.entries()) {
+            const div = document.createElement('div');
+
+            let child = document.createElement('div');
+            child.innerHTML = (it + 1);
+            div.appendChild(child);
+
+            child = document.createElement('div');
+            child.innerHTML = form.formTitle;
+            div.appendChild(child);
+
+            child = document.createElement('div');
+            child.innerHTML = form.hrEmployer;
+            div.appendChild(child);
+
+            let right = 0;
+            const all = form.points.length;
+
+            for (const point of form.points) {
+                if (point) right++;
+            }
+
+            child = document.createElement('div');
+            child.innerHTML = `${((right / all) * 100).toFixed(0)}%`;
+            div.appendChild(child);
+
+            child = document.createElement('div');
+
+            const img = new Image();
+            img.src = eyePng;
+            img.onclick = () => {
+                this.show(form);
+            };
+            child.appendChild(img);
+
+            div.appendChild(child);
+            $id('checkedForm-list-table').appendChild(div);
+        }
+
+        this.showAll();
+
+        $id('checkedForm-content-loading').remove();
+    },
+
+    open() {
+        if (!this.initialized)
+            this.init();
+        this.showAll();
+    },
+
+    showAll() {
+        $id('checkedForm-list').style.display = 'block';
+        $id('checkedForm-results').style.display = 'none';
+    },
+
+    show(which) {
+        $id('checkedForm-list').style.display = 'none';
+        $id('checkedForm-results').style.display = 'block';
+
+        $id('checkedForm-results-loading').style.display = 'block';
+        $id('checkedForm-results-container').style.display = 'none';
+
+        // TODO: bazka
```

Comment:
Pobieranie z bazy drugi raz? Czego?

Reply:
No tutaj mogło się wkraść nieporozumienie, bo to co mi wysłałeś nie miało pytań tylko odpowiedzi, więc uznałem że tego nie ma i trzeba formę pobrać z bazy...

Reply:
Ok, rozumiem że oprócz odpowiedzi wyświetla formularz, jak tak to ok